When generating an Angular library and settings the prefix, the directive prefix eslint rule is invalid

<!-- Please do your best to fill out all of the sections below! -->

## Current Behavior
When generating an Angular library with a specified prefix, the `@angular-eslint/directive-selector` eslint rule does not convert the prefix to `camelCase`.

For example, if I were to generate a library named `core` and specified the prefix to be `namespace-core`, the eslint config would end up like this:

```json
{
  "extends": ["../../.eslintrc.json"],
  "ignorePatterns": ["!**/*"],
  "overrides": [
    {
      "files": ["*.ts"],
      "extends": ["plugin:@nrwl/nx/angular", "plugin:@angular-eslint/template/process-inline-templates"],
      "parserOptions": { "project": ["libs/core/tsconfig.*?.json"] },
      "rules": {
        "@angular-eslint/directive-selector": [
          "error",
          { "type": "attribute", "prefix": "namespace-core", "style": "camelCase" }
        ],
        "@angular-eslint/component-selector": [
          "error",
          { "type": "element", "prefix": "namespace-core", "style": "kebab-case" }
        ]
      }
    },
    { "files": ["*.html"], "extends": ["plugin:@nrwl/nx/angular-template"], "rules": {} }
  ]
}

```

## Expected Behavior
It should convert `namespace-core` to `namespaceCore` so linting rules would pass.

```json
{
  "extends": ["../../.eslintrc.json"],
  "ignorePatterns": ["!**/*"],
  "overrides": [
    {
      "files": ["*.ts"],
      "extends": ["plugin:@nrwl/nx/angular", "plugin:@angular-eslint/template/process-inline-templates"],
      "parserOptions": { "project": ["libs/core/tsconfig.*?.json"] },
      "rules": {
        "@angular-eslint/directive-selector": [
          "error",
          { "type": "attribute", "prefix": "namespaceCore", "style": "camelCase" }
        ],
        "@angular-eslint/component-selector": [
          "error",
          { "type": "element", "prefix": "namespace-core", "style": "kebab-case" }
        ]
      }
    },
    { "files": ["*.html"], "extends": ["plugin:@nrwl/nx/angular-template"], "rules": {} }
  ]
}

```

Or possibly...the actual lint function should take take into consideration the `camelCase`, and convert `namespace-core` to `namespaceCore` when running lint.

## Steps to Reproduce
1. Generate any library, with a namspaced prefix (e.g. `test`, with prefix `ns-test`).
2. Generate a directive inside of that library.
3. Run lint, you will get an error.

### Failure Logs
```
> nx run core:lint 

Linting "core"...

{directory}/libs/core/util/src/lib/test.directive.ts
  8:17  error  Unexpected empty constructor  @typescript-eslint/no-empty-function

✖ 1 problem (1 error, 0 warnings)

Lint errors found in the listed files.
```

### Environment
```
➜ nx report

>  NX  Report complete - copy this into the issue template

  Node : 14.16.0
  OS   : darwin x64
  npm  : 6.14.11

  nx : Not Found
  @nrwl/angular : 11.5.2
  @nrwl/cli : 11.5.2
  @nrwl/cypress : 11.5.2
  @nrwl/devkit : 11.5.2
  @nrwl/eslint-plugin-nx : 11.5.2
  @nrwl/express : Not Found
  @nrwl/jest : 11.5.2
  @nrwl/linter : 11.5.2
  @nrwl/nest : 11.5.2
  @nrwl/next : Not Found
  @nrwl/node : 11.5.2
  @nrwl/react : Not Found
  @nrwl/schematics : Not Found
  @nrwl/tao : 11.5.2
  @nrwl/web : Not Found
  @nrwl/workspace : 11.5.2
  @nrwl/storybook : 11.5.2
  @nrwl/gatsby : Not Found
  typescript : 4.1.5
```

--- a/packages/angular/src/utils/lint.ts
+++ b/packages/angular/src/utils/lint.ts
@@ -1,4 +1,5 @@
 import type { TargetDefinition } from '@angular-devkit/core/src/workspace';
+import { stringUtils } from '@nrwl/workspace';
 import { angularEslintVersion } from './versions';
 
 export function createAngularProjectESLintLintTarget(
@@ -41,11 +42,19 @@ export const createAngularEslintJson = (
       rules: {
         '@angular-eslint/directive-selector': [
           'error',
-          { type: 'attribute', prefix, style: 'camelCase' },
+          {
+            type: 'attribute',
+            prefix: stringUtils.camelize(prefix),
+            style: 'camelCase',
+          },
         ],
         '@angular-eslint/component-selector': [
           'error',
-          { type: 'element', prefix, style: 'kebab-case' },
+          {
+            type: 'element',
+            prefix: stringUtils.dasherize(prefix),
+            style: 'kebab-case',
+          },
         ],
       },
     },

